PF1/Assignment4/ProcessScheduling.java

```diff
@@ -0,0 +1,131 @@
+package assignment4;
+
+//Process scheduling using First Come First Serve(FCFS)
+public class ProcessScheduling {
+	public boolean isArrayEqual(int[][] expected, int[][] actual){
+		boolean flag= true;
+				
+		for(int row=0; row< actual.length; row++){
+			for(int column=0; column< actual[0].length; column++){
+				if(actual[row][column] == expected[row][column]){
+					flag= false;
+					break;
+				}
+			}
+		}
+		return flag;
+	}
+	 
+	public int[][] FCFS( int arrival_time[], int job_size[] ) 
+	{
+
+	int i,j,start_time=1;
+	 int result_array[][]=new int[arrival_time.length][5];
```

Comment:
Dont use _ with variable name, _ is used for constants. That should be like resultArray.